container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -319,9 +319,16 @@ function run_git_tempest {
     pushd $TEMPEST_DIR
 
     discover-tempest-config ${TEMPESTCONF_ARGS} ${TEMPESTCONF_OVERRIDES} \
+    && tempest cleanup --init-saved-state || true \
     && tempest run ${TEMPEST_ARGS}
     RETURN_VALUE=$?
 
+    # Run tempest cleanup to delete any leftover resources
+    # when not in debug mode
+    if [ ${TEMPEST_DEBUG_MODE} == false ]; then
+        tempest cleanup || true
```

Comment:
I think it is ok to have the command without `|| true` here. The script will continue execution even when the command fails. The failure of the whole pod is defined by RETURN_VALUE=$?.